todo/src/actions/actions.js

```diff
@@ -0,0 +1,35 @@
+import uuid from 'uuid';
+
+export const ADD_TODO = 'ADD_TODO';
+export const TOGGLE_COMPLETED = 'TOGGLE_COMPLETED';
+export const DELETE_TODO = 'DELETE_TODO';
+export const DELETE_COMPLETED = 'DELETE_COMPLETED';
+
+export const addTodo = todo => {
+  return {
+    type: ADD_TODO,
+    payload: {
+      id: uuid.v4(),
```

Comment:
good job doing this, and doing it before it gets to the reducer -- remember there should be zero randomized side-effects within your reducers, which includes anything randomized with Date.now(), Math.random(), or a randomizer library like uuid